Documentation/ceph-quickstart.md

```diff
@@ -18,12 +18,7 @@ Kubernetes **v1.11** or higher is supported by Rook.
 
 ## Prerequisites
 
-To make sure you have a Kubernetes cluster that is ready for `Rook`, you can [follow these instructions](k8s-pre-reqs.md).
-
-In order to configure the Ceph storage cluster, at least one of these local storage options are required:
-- Raw devices (no partitions or formatted filesystems)
-- Raw partitions (no formatted filesystem)
-- PVs available from a storage class in `block` mode
+Please refer to [the document of prerequisites](ceph-prerequisites.md)
```

Comment:
The requirement for disks is important enough that I think the duplication is ok so the make sure to not skip over that requirement. All the other pre-requisites are either optional or can be fixed later.

Reply:
OK, I'll remove this change in the next push.